lib/guardian/user_guardian.rb

```diff
@@ -224,4 +224,11 @@ def can_delete_user_associated_accounts?(user)
   def can_change_tracking_preferences?(user)
     (SiteSetting.allow_changing_staged_user_tracking || !user.staged) && can_edit_user?(user)
   end
+
+  def can_create_theme?(admin)
+    # this modifier is used to further restrict theme creation, it's not
+    # possible to use this modifier to open up theme creation permissions (e.g.
+    # to non-admins)
+    DiscoursePluginRegistry.apply_modifier(:user_guardian_can_create_theme, true, admin, self)
```

Comment:
```suggestion
  def can_create_theme?(user)
    return false if !user.admin?
    # this modifier is used to further restrict theme creation, it's not
    # possible to use this modifier to open up theme creation permissions (e.g.
    # to non-admins)
    DiscoursePluginRegistry.apply_modifier(:user_guardian_can_create_theme, true, user, self)
```

I think it is better to ensure this method will always return `false` for a non admin user.